src/backends/policy_engine/souffle/testdata/simple_failing_sql.ir

```diff
@@ -1,10 +1,10 @@
 module m0 {
   block b0 {
-    %0 = sql.merge [control_input_start_index: 0]()
-    %1 = sql.tag_transform [rule_name: set_restricted] (%0.out)
-    %2 = sql.tag_transform [rule_name: set_ssn_integrity] (%1.out)
+    %0 = sql.merge [control_input_start_index: "0"]()
```

Comment:
This is a temporary fix to pass test. This PR#594 handles int attributes reverting this change.